server/kubeconfig.go

```diff
@@ -101,18 +101,24 @@ func (s *kubeConfigServer) GetOrganizationSetting(ctx context.Context, req *sent
 	if err != nil {
 		return nil, err
 	}
+	sessData, ok := service.GetSessionDataFromContext(ctx)
+	if !ok {
+		return nil, fmt.Errorf("unable to retrieve session data")
+	}
+
 	if orgID != opts.Organization {
 		opts.Organization = orgID
 	}
-	ks, err := s.kss.Get(ctx, opts.Organization, "", false)
+	ks, err := s.kss.Get(ctx, opts.Organization, sessData.Account, false)
```

Comment:
why are we passing account here? this should be organization specific setting and not account specific ..